src/utils/index.js

```diff
@@ -17,4 +18,21 @@ export function renderWithRedux(
   };
 }
 
+/**
+ * Returns the array of tweets sorted by date
+ * @param tweetsArr: Array of tweets
+ * @param order: Order direction
+ * @returns {Object}
+ */
+
+export function sortTweetsByDate(tweetsArr = [], order = DESC_ORDER) {
+  let sortedTweets = [...tweetsArr]
+
+  return sortedTweets.sort(function(a,b){
```

Comment:
Credits go to author:
https://stackoverflow.com/questions/10123953/how-to-sort-an-array-by-a-date-property